wallet/src/Pos/Wallet/Web/Methods/Payment.hs

```diff
@@ -128,6 +131,20 @@ instance
     getNewAddress (accId, passphrase) = do
         clientAddress <- L.newAddress RandomSeed passphrase accId
         decodeCTypeOrFail (cadId clientAddress)
+    -- We are creating a sample HD address here. It has the same
+    -- length (76) as the biggest one created by Daedalus.
+    getFakeChangeAddress = do
+        let passphrase = DBA.replicate 10 255
+        SecretKey sk <- snd <$> keyGen
+        encSK <- mkEncSecret passphrase sk
+        case deriveLvl2KeyPair
```

Comment:
I don't think it's acceptable, `deriveLvl2KeyPair` adds redundant overhead for creating tx. also it can generate different size for different addresses because it depends on secret key

Reply:
> I don't think it's acceptable, deriveLvl2KeyPair adds redundant overhead for creating tx.

Do you mean performance overhead or what? I don't think it will be big overhead, tx creation has a lot of logic. And `deriveLvl2KeyPair` is almost always used once anyway.
Do you propose to just use hardcoded constant address?

> also it can generate different size of addresses because it depends on secret key

Yes, I realized it today already, will fix.

Reply:
I realized that we do a lot of `deriveLvl2KeyPair` during building of a tx and one more won't add performance overhead